### ISSUE TITLE
Bug with case-sensitive speaker ID causing "NotImplementedError" errors coming from the qwen_tts interface

### ISSUE THREAD

**Issue by MrMindBlow** [FranckyB/Voice-Clone-Studio#15]
Hello Francois

Great project, been loving it thanks to its intuitive interface and I have been dissecting it recently as I stumbled upon an issue.

When training a new model with a mixed-case speaker name (I used "Test_Man"), the sft_12hz.py training script saves the speaker ID into config.json preserving the case which is working as designed but the issue I ran into is that the underlying qwen_tts inference library enforces case-insensitivity by **lowercasing** the speaker name **before** lookup. This results in a "_NotImplementedError_" when trying to use the trained model in Voice Presets, because the inference engine looks for "test_man" but the config contains "Test_Man".

I found two ways to fix this:
1) Edit the config directly so that it says "test_man" instead of "Test_Man" which works but will break if I train another model
or
2) Update sft_12hz.py to normalize the speaker name to lowercase when saving the configuration which will work on any and all further models

Current implementation:

```
talker_config["spk_id"] = {args.speaker_name: 3000}

talker_config["spk_is_dialect"] = {args.speaker_name: False}
```

My suggestion (Fix 2):

```
talker_config["spk_id"] = {args.speaker_name.lower(): 3000}

talker_config["spk_is_dialect"] = {args.speaker_name.lower(): False}
```

Small thing! :)
I would have forked, fixed and PR'ed it but I think submitting an Issue is enough as the fix is quite quick to implement.

**Comment by FranckyB** [comment]
Fixed... Even smaller solution.. 
Simply added the .lower() when sending to finetune from my code.
